sycl/source/detail/scheduler/graph_builder.cpp

```diff
@@ -508,10 +508,10 @@ Scheduler::GraphBuilder::findDepsForReq(MemObjRecord *Record,
   std::vector<Command *> Visited;
   const bool ReadOnlyReq = Req->MAccessMode == access::mode::read;
 
-  std::vector<Command *> ToAnalyze{std::move(Record->MWriteLeaves.toVector())};
+  std::vector<Command *> ToAnalyze{Record->MWriteLeaves.toVector()};
 
   if (!ReadOnlyReq) {
-    std::vector<Command *> V{std::move(Record->MReadLeaves.toVector())};
+    std::vector<Command *> V{Record->MReadLeaves.toVector()};
```

Comment:
These are to correct this:
graph_build.cpp: error: moving a temporary object prevents copy elision [-Werror,-Wpessimizing-move]
 std::vector<Command *> ToAnalyze{std::move(Record->MWriteLeaves.toVector())};
                 error: moving a temporary object prevents copy elision [-Werror,-Wpessimizing-move]
    std::vector<Command *> V{std::move(Record->MReadLeaves.toVector())};